root/Makefile

```diff
@@ -96,9 +97,15 @@ pre-commit: fmt
 .PHONY: build
 build:: pre-build gobuild
 
+## lint:            run code linters
+.PHONY: lint
+# Note: We run pre-test for compat. Remove on the next breaking change.
```

Comment:
definitely make a ticket for this and add it to the breaking changes dashboard julia has. this one has a high chance of being forgotten about